src/components/shared/transactionsTable/index.js

```diff
@@ -143,14 +144,15 @@ class TransactionsTable extends React.Component {
 
   render() {
     const {
-      title, transactions, columns, isLoadMoreEnabled, t,
+      title, transactions, columns, isLoadMoreEnabled, t, filters, isFilterDropdownExpandable,
```

Comment:
I think `isFilterDropdownExpandable` is not needed to be passed from outside. IMO `isFilterDropdownExpandable = filters.length > 3` and is something that can be only inside the shared https://github.com/LiskHQ/lisk-hub/blob/development/src/components/shared/filterDropdownButton/index.js